scripts/coverage_report_jenkins.py

```diff
@@ -199,7 +205,13 @@ def merge_coverage_data(ssh, coverage_dir):
     # The sonar-scanner will actually need the .resultset.json file it
     # uses to be in /coverage/.resultset.json (i.e. the root of the coverarage
     # directory), so lets create a symlink:
-    ssh.run_command('ln -s merged/.resultset.json {}/.resultset.json'.format(coverage_dir))
+    merged_resultset = '{}/merged/.resultset.json'.format(coverage_dir)
```

Comment:
In general, I'd love to see use of `py.path` in this tooling, instead of string path manipulation and directly implemented bash for delete-if-exists as below.

http://py.readthedocs.io/en/latest/path.html

Reply:
Though this might only really be applicable to use of local paths, unfortunately.

Reply:
It appears to be applicable, but of course if I check for paths existence and such using the object that does not work.